core/src/main/java/org/mule/runtime/core/internal/processor/strategy/util/ProfilingUtils.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.core.internal.processor.strategy.util;
+
+import org.mule.runtime.api.component.Component;
+import org.mule.runtime.api.component.location.ComponentLocation;
+import org.mule.runtime.core.api.MuleContext;
+import org.mule.runtime.core.api.processor.ReactiveProcessor;
+import org.mule.runtime.core.internal.processor.chain.InterceptedReactiveProcessor;
+import org.mule.runtime.core.privileged.processor.chain.HasLocation;
+
+/**
+ * Utility methods for handling profiling of processing strategy.
+ * <p>
+ * MULE-19594: Handle expression problem for Reactive Processor (ComponentLocation)
```

Comment:
what does this mean?

Reply:
Changed to "Problem for retrieving component location from generic reactive processor." 
https://en.wikipedia.org/wiki/Expression_problem